### PR TITLE
Bump readerSdkVersion 1.7.6 and targetSdkVersion to 34

### PR DESCRIPTION
**Pull request by JonathanNakhla-Square**
- Update readerSdkVersion to 1.7.6
- Update targetSdkVersion to 34
- Update gradle version to 8.6
- Update quick start guide link

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,12 +1,11 @@
 apply plugin: 'com.android.application'
 
 android {
-  compileSdkVersion 33
-  buildToolsVersion "30.0.3"
```
Comment (fka3) [inline app/build.gradle:5]: Where did (the logic of) this line go?

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -1,6 +1,4 @@
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="com.example.readersdk">
```
Comment (fka3) [inline app/src/main/AndroidManifest.xml:3]: This becomes the `namespace` in build.gradle, right?

File: gradle.properties [hunk 3]
```diff
@@ -3,11 +3,14 @@ org.gradle.jvmargs=-Xmx4g
 ## Uncomment the properties below and set their correct value.
 
 ## The Application ID displayed on the Credentials tab in the Square Application Control Panel.
-## SQUARE_READER_SDK_APPLICATION_ID=
+SQUARE_READER_SDK_APPLICATION_ID=sq0idp-tSv1iVAOJUzfzK3eUvATFg
```
Comment (fka3) [inline gradle.properties]: - [x] Don't commit these testing values.  We shold also rotate the repository password 'cause you pushed it. ;-)

**Review by fka3 — APPROVED** [review]

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (JonathanNakhla-Square) [reply app/src/main/AndroidManifest.xml:3]: Yep correct.

File: gradle.properties (hunk 3 above)
Comment (JonathanNakhla-Square) [reply gradle.properties]: Removed and invalidated the password!

File: app/build.gradle (hunk 1 above)
Comment (JonathanNakhla-Square) [reply app/build.gradle:5]: It's deprecated and no longer needed as the gradle plugin behind the scenes now knows which build tools version to use. This is all per [this documentation](https://developer.android.com/tools/releases/build-tools):

>If you're using [Android plugin for Gradle 3.0.0](https://developer.android.com/studio/releases/gradle-plugin#3-0-0) or higher, your project automatically uses a default version of the build tools that the plugin specifies.

**Pull request merged**